build-scripts/generateResources.js

```diff
@@ -166,10 +167,11 @@ var createResjson = function(task, taskPath) {
 }
 
 function addVersionToTask(task, versionInfo) {
+    versionInfo = semverParse(versionInfo)
```

Comment:
I'd rather do a string.split than pulling in another package since we only need to do it once and it is in a known format

Reply:
done